xds/src/test/java/io/grpc/xds/XdsNameResolverProviderTest.java

```diff
@@ -91,22 +91,22 @@ public void newNameResolver() {
   public void validName_withAuthority() {
     XdsNameResolver resolver =
         provider.newNameResolver(
-            URI.create("xds-experimental://trafficdirector.google.com/foo.googleapis.com"), args);
+            URI.create("xds://trafficdirector.google.com/foo.googleapis.com"), args);
     assertThat(resolver).isNotNull();
     assertThat(resolver.getServiceAuthority()).isEqualTo("foo.googleapis.com");
   }
 
   @Test
   public void validName_noAuthority() {
     XdsNameResolver resolver =
-        provider.newNameResolver(URI.create("xds-experimental:///foo.googleapis.com"), args);
+        provider.newNameResolver(URI.create("xds:///foo.googleapis.com"), args);
     assertThat(resolver).isNotNull();
     assertThat(resolver.getServiceAuthority()).isEqualTo("foo.googleapis.com");
   }
 
   @Test
   public void invalidName_hostnameContainsUnderscore() {
-    URI uri = URI.create("xds-experimental:///foo_bar.googleapis.com");
+    URI uri = URI.create("xds:///foo_bar.googleapis.com");
```

Comment:
No tests will verify that xds-experimental still works?

Reply:
We can copy the test class as well. @ericgribkoff The interop test is testing explicitly with xds-experimental right?

Reply:
Not sure it's worth copying the whole test class. Was thinking about that one test here could verify that the `xds-experimental` provider at least exists but I guess that might be awkward to reference `XdsExperimentalNameResolverProviderTest.java` since this is `XdsNameResolverProviderTest`.

Yes, the interop testing is explicitly using xds-experimental for now. But we could actually switch it in this PR on master: the scheme is specified in `https://github.com/grpc/grpc-java/blob/master/buildscripts/kokoro/xds.sh`. 

Are we really only keeping around `xds-experimental` for interop testing and plan to have it fully removed in the next release? Or will there be some overlap period where users can specify `xds` or `xds-experimental`?

Reply:
> Not sure it's worth copying the whole test class. 

Copying is simple and it's super easy to remove when it's ready to.

Reply:
I guess we can directly do the swap. The scheme is sort of arbitrary and the only impact is the resolved being loaded in gRPC client. Seems no value to do a transition period.

Reply:
> Yes, the interop testing is explicitly using xds-experimental for now. But we could actually switch it in this PR on master: the scheme is specified in https://github.com/grpc/grpc-java/blob/master/buildscripts/kokoro/xds.sh

I thought the scheme xds-experimental is specified in some other repo than grpc-java. Will change it in https://github.com/grpc/grpc-java/blob/master/buildscripts/kokoro/xds.sh as well.

>Are we really only keeping around xds-experimental for interop testing and plan to have it fully removed in the next release? Or will there be some overlap period where users can specify xds or xds-experimental?

I don't know. @srini100 Do we want to keep the 'xds-experimental' scheme working for our customers who are testing against master or next releases (1.29, 1.30) for a while?

Reply:
@ericgribkoff Does changing https://github.com/grpc/grpc-java/blob/master/buildscripts/kokoro/xds.sh with 'xds' scheme affect other languages? I mean is there java server against go client and vice versa?

Reply:
We talked about keeping xds-experimental for at least one more release (1.29.0) and remove it in 1.30.0. This should be same across other langs so discuss with other lang owners too.